sdk/purview/azure-purview-workflow/azure/purview/workflow/_client.py

```diff
@@ -102,7 +102,7 @@ def __init__(self, endpoint: str, credential: "TokenCredential", **kwargs: Any)
         self.approval = ApprovalOperations(self._client, self._config, self._serialize, self._deserialize)
         self.task_status = TaskStatusOperations(self._client, self._config, self._serialize, self._deserialize)
 
-    def send_request(self, request: HttpRequest, **kwargs: Any) -> HttpResponse:
+    def send_request(self, request: HttpRequest, *, stream:bool=False, **kwargs: Any) -> HttpResponse:
         """Runs the network request through the client's chained policies.
```

Comment:
Most errors were of this variety:
`azure/purview/workflow/aio/operations/_operations.py:423: [C4758(docstring-keyword-should-match-keyword-only), WorkflowOperations.create_or_replace] "content_type" missing in docstring or in method signature. There should be a direct correlation between :keyword: arguments in the docstring and keyword-only arguments in method signature. See details: https://azure.github.io/azure-sdk/python_documentation.html#docstrings
`

The approach I took was to add docstring keyword arg as a specific keyword arg in the method parameters. In the case of this stream keyword arg it needs to be passwed to send_request.